### PR TITLE
Add Multi-Regions Support

### PR DESCRIPTION
**Pull request by noahjohnhay**

### CODE REVIEW

File: src/args.rs [hunk 1]
```diff
@@ -1,12 +1,11 @@
+use rusoto_core::Region;
+
 /// This tool does bulk operations against sub-components in a cluster. Use with great care.
 #[derive(Debug, StructOpt)]
 pub struct Args {
     /// AWS profile for authentication
     #[structopt(long = "profile")]
     pub profile: Option<String>,
-    /// AWS region of cluster
```
Comment (jonathanmorley) [inline src/args.rs]: These comments get converted into help text for the CLI, we should avoid removing them

Comment (noahjohnhay) [reply src/args.rs]: oh my bad want me to add that back now?

Comment (jonathanmorley) [reply src/args.rs]: please, for each of the new region parameters

Comment (jonathanmorley) [reply src/args.rs]: looks good, will merge after checks

**Pull request merged**